### DISCUSSION TITLE
Organization access - Access request pending

### DISCUSSION THREAD

**Discussion by YagneshAPatel** [community/community#33742] [New to GitHub]
Hi Team,
I have started using Git recently and getting familiar with Git Flow.  I am using Pycharm for my dev work and found that using PyCharm Commit+Push is not working in a single step.  From Command line Push is working fine but from Window Popup it is not working. 

![Screenshot 2022-09-21 161619](https://user-images.githubusercontent.com/111067366/191602436-9bc3c0f7-014d-471b-9987-2f379c13ed38.png)

on **GitHub - Profile - Integration - Application**, It is showing 

**Organization access - Access request pending** 

for my company.  

I am not sure why and how to fix this issues. For other applications like GitHub Desktop and Visual Studio; it is not showing such thing.
Can someone help in this or share what could be wrong in setup.  

Regards.

**Comment by queenofcorgis** [comment]
Hi there! The best place to go for answers to account-specific questions like this one is by opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Reply by Abhi0806** [reply]
Did you found the solution for this issue ?

**Comment by RuoyuLiu-f** [comment]
You can look for third-party sources
